cmd/speedway-cli/speedwaycmd/schema/create.go

```
@@ -55,44 +57,35 @@ func bootstrapCreateSchemaCommand(ctx context.Context, logger *golog.Logger) (cr
 
 			fields := make(map[string]types.SchemaKind)
 			if path, err := cmd.Flags().GetString("file"); err == nil && path == "" {
-				prompt := promptui.Prompt{
-					Label: "Enter your Schema Fields",
+				// Prompt the user for a list of label:field to create a schema
+				// label:field is a comma separated string
+				// e.g. "name:string,age:int"
+				fieldsPrompt := promptui.Prompt{
+					Label: "Enter a list (sperated by commas) of label:kind for the schema",
 				}
 
-				var repeat bool
-				for !repeat {
-					// make schemaFields []string
-					schemaField, err := prompt.Run()
-					if err != nil {
-						fmt.Printf("Command failed %v\n", err)
-						return
-					}
-					// for every schemaFields, prompt for the type of the field
-					selectSchemaKind := promptui.Select{
-						Label: "Select a Schema Field",
-						Items: []string{
-							"LIST",
-							"BOOL",
-							"INT",
-							"FLOAT",
-							"STRING",
-							"BYTES",
-							"LINK",
-						},
-					}
-					_, result, err := selectSchemaKind.Run()
-					if err != nil {
-						fmt.Printf("Prompt failed %v\n", err)
+				result, err := fieldsPrompt.Run()
+				if err != nil {
+					logger.Fatalf(status.Error("Command failed"), err)
+					return
+				}
+
+				// Parse the result into a types.Schema
+				for _, field := range strings.Split(result, ",") {
+					fieldSplit := strings.Split(field, ":")
+					if len(fieldSplit) != 2 {
+						logger.Fatalf(status.Error("Invalid field format"), err)
 						return
 					}
-					sk := utils.ConvertSchemaKind(result)
-					fields[schemaField] = sk
-					repeat = prompts.QuitSelector("Create another Schema Field?")
+					// take the second element of the split and convert it to a types.SchemaKind
+					kind := utils.ConvertSchemaKind(fieldSplit[1])
```

convert might also need to return an error now because kind input is arbitrary.